metrics_utility/management/validation.py

```diff
@@ -107,6 +139,190 @@ def handle_crc_ship_target():
     return billing_provider_params
 
 
+def validate_report_type(errors, method):
+    """
+    Validates the 'METRICS_UTILITY_REPORT_TYPE' environment variable against a set of valid report types.
+
+    If the environment variable is set and its value is not in the list of valid report types,
+    an error message is appended to the provided errors list.
+
+    Args:
+        errors (list): A list to which error messages will be appended if validation fails.
+
+    Returns:
+        str or None: The value of the 'METRICS_UTILITY_REPORT_TYPE' environment variable if set, otherwise None.
+    """
+    report_type = os.getenv('METRICS_UTILITY_REPORT_TYPE', None)
+    if report_type and report_type not in VALID_REPORT_TYPES:
+        errors.append(
+            f'Invalid METRICS_UTILITY_REPORT_TYPE: {report_type}. Valid values: {", ".join(VALID_REPORT_TYPES)}. '
+            f'Please note these values are case sensitive'
+        )
+        if method == 'build' and report_type is None:
+            errors.append(
+                f'Invalid METRICS_UTILITY_REPORT_TYPE is Empty. Valid values: {", ".join(VALID_REPORT_TYPES)}. '
+                f'Please note these values are case sensitive'
+            )
```

Comment:
```suggestion
    if method == 'build' and report_type is None:
        errors.append(
            f'Invalid METRICS_UTILITY_REPORT_TYPE is Empty. Valid values: {", ".join(VALID_REPORT_TYPES)}. '
            f'Please note these values are case sensitive'
        )
```

(inside the top if, report_type is never None)